doc/wiki/User-Guide/Projects/Subproject.md

```diff
@@ -52,6 +54,30 @@ View all subprojects where the current user has view-permissions on.
 
 4. At the bottom half of the site all subprojects of the current project are viewed
 
+## Filter subprojects
+
+**Description:**
+
+There are 3 ways how projects can be filtered on the overview page:
+
+1. Searchbar
+2. URL
+
+The Searchbar can be used to search all projects for a term included in name, tag or status. Prefixes can be used to specify the search context (e.g. tag:mycustomtag). If no prefix is used display name, tag and status is searched for a match. After typing the project list and the URL are instantly updated. The URL can then be shared to other users including the filter.
```

Comment:
Prefixes can be used.... -> These can also be used as prefixes to ...
is searched -> are searched
instantly updated -> updated instantly

Reply:
done - also done for Projects.md